### ISSUE TITLE
ignore_errors options broken

### ISSUE THREAD

**Issue by owais** [python/mypy#2872]
I wanted to try mypy on a project and wanted to ignore some modules but `ignore_errors` configuration option does not work for some reason.  mypy complains that `ignore_errors` is not a valid option.

```bash
$ mypy main.py                                                                                                                            0|23:55:25
mypy.ini: [mypy]: Unrecognized option: ignore_errors = True
```

mypy.ini
```ini
[mypy]
ignore_errors = True
```

```main.py
print('hello world')
```

## Steps to reproduce
1. Create `main.py` and `mypy.ini` from above.

2. 
```bash
$ python3.6 -m venv venv
$ venv/bin/pip install mypy-lang
$ venv/bin/mypy main.py
```

**Comment by gvanrossum** [comment]
Sorry, we've renamed the PyPI name of the project to simply "mypy". The mypy-lang project is an old version that doesn't support this flag, and never will (it is frozen at 0.4.6). Try again:
```
venv/bin/pip uninstall mypy-lang
venv/bin/pip install mypy
```
